app/services/dttp/contact_service/create.rb

```diff
@@ -12,7 +12,7 @@ def initialize(trainee:)
       end
 
       def call
-        response = Client.post(endpoint_path, { body: body,
+        response = Client.post(endpoint_path, { body: trainee.to_dttp_params.as_json,
```

Comment:
Just for clarity is the presenter for DTTP or for View as well?

Reply:
Just for DTTP concerns